calico/felix/frules.py

```diff
@@ -148,9 +159,10 @@ def install_global_rules(config, v4_filter_updater, v6_filter_updater,
             },
             {
                 CHAIN_FORWARD: set([CHAIN_FROM_ENDPOINT, CHAIN_TO_ENDPOINT]),
-                CHAIN_INPUT: set([CHAIN_FROM_ENDPOINT]),
-            },
-            async=False)
+                CHAIN_INPUT: set(),
+            }
+        )
+
```

Comment:
You removed "async=False" here. Why was that?

In practice I think it is benign (you put some stuff on the iptables updater queue, then put something else on and wait for it to be processed - so it doesn't matter because when you wait for the second, the first must have been completed). However, less confusing to leave it as it was.